release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2879,6 +2879,49 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Previously, the `vmware-vsphere-csi-driver-operator` Container Storage Interface (CSI) driver entered panic mode when the {vmw-full} vCenter address was incorrect or missing. With this release, the CSI driver does not go into panic mode if the {vmw-full} vCenter address is incorrect or missing. (link:https://issues.redhat.com/browse/OCPBUGS-52207[*OCPBUGS-52207*])
+
+* Previously, the *Observe* section on the web console did not show items contributed from plugins unless certain flags related to the monitoring were set. However, these flags prevented other plugins, such as logging, distributed tracing, network observability, and so on, from adding items to the *Observe* section. With this release, the monitoring flags are removed so that other plugins can add items to the *Observe* section. (link:https://issues.redhat.com/browse/OCPBUGS-52205[*OCPBUGS-52205*])
+
+* Previously, the *Cluster Settings* page would not properly render during a cluster update if the `ClusterVersion` did not receive a `Completed` update.  With this release, the *Cluster Setting* page properly renders even if the `ClusterVersion` has not received a `Completed` update. (link:https://issues.redhat.com/browse/OCPBUGS-51292[*OCPBUGS-51292*])
+
+* Previously, the alerts links on the *Alert rules* page of the *Developer* perspective included external labels to invalid links. This happened because the URL for the *Alerts* page did not expect external labels. With this release, the external labels are no longer added to the alerts URL on the *Alert rules* page so the alerts links are accurate. (link:https://issues.redhat.com/browse/OCPBUGS-51126[*OCPBUGS-51126*])
+
+* Previously, users could not create `NodePools` resources with ARM64 architecture on non-AWS or {azure-short} platforms. This bug resulted in validation errors that prevented the addition of bare-metal compute nodes and caused CEL validation blocks when creating a `NodePool` resource. The fix modifies the `NodePool` spec validation rules to allow ARM64 architecture on non-AWS or {azure-short} or Agent platforms by setting `None` for the `self.platform.type` section, so users can now create `NodePools` with ARM64 architecture specifications on non-AWS or {azure-short} bare-metal platforms, expanding functionality for end users. (link:https://issues.redhat.com/browse/OCPBUGS-46440[*OCPBUGS-46440*])
+
+* Previously, for a `kubevirt-csi` pod that ran on a node in a guest cluster, the persistent volume claim (PVC) from the guest cluster was removed from the VM after the VM was rebooted. However, the `VolumeAttachment` resource was not removed and this caused issues for the cluster as it expected the PVC to be attached to the VM. With this release, after a VM is rebooted, the `VolumeAttachment` resource is removed so that the cluster issues no longer occur. (link:https://issues.redhat.com/browse/OCPBUGS-44623[*OCPBUGS-44623*])
```

Comment:
```suggestion
* Previously, for a `kubevirt-csi` pod that ran on a node in a guest cluster, the persistent volume claim (PVC) from the hosted cluster was removed from the virtual machine (VM) after the VM was restarted. However, the `VolumeAttachment` resource was not removed and this caused issues for the cluster as it expected the PVC to be attached to the VM. With this release, after a VM is rebooted, the `VolumeAttachment` resource is removed so that the cluster issues no longer occur. (link:https://issues.redhat.com/browse/OCPBUGS-44623[*OCPBUGS-44623*])
```

We use the term _**hosted cluster**_ rather than _**guest cluster**_ . Both terms mean the same.